bundles/org.corpus_tools.hexatomic.core/src/main/java/org/corpus_tools/hexatomic/core/ui/AboutDialog.java

```diff
@@ -0,0 +1,176 @@
+  }
+
+  /**
+   * Create the user documentation URL for the current version.
+   * 
+   * @return The complete URL.
+   */
+  public static String getOnlineDocumentationUrl() {
+    return "https://hexatomic.github.io/hexatomic/user/v" + getShortVersion() + "/";
+  }
+
+  /**
+   * Create contents of the dialog.
+   * 
+   * @param parent The parent
+   */
+  @Override
+  protected Control createDialogArea(Composite parent) {
+    Composite container = (Composite) super.createDialogArea(parent);
+
+    Label lblHeader = new Label(container, SWT.NONE);
+    lblHeader.setLayoutData(new GridData(SWT.CENTER, SWT.CENTER, true, false, 1, 1));
+    lblHeader.setText("Hexatomic");
+    lblHeader.setFont(headerFont);
+
+    Label lblVersion = new Label(container, SWT.NONE);
+    lblVersion.setLayoutData(new GridData(SWT.CENTER, SWT.CENTER, false, false, 1, 1));
+    lblVersion.setText("Version: " + getFullVersion());
+    lblVersion.setFont(versionFont);
+
+    Link lnkHomepage = new Link(container, SWT.NONE);
+    lnkHomepage.setLayoutData(new GridData(SWT.CENTER, SWT.CENTER, false, false, 1, 1));
+    lnkHomepage.setText("Homepage: <a>https://hexatomic.github.io/</a>");
+    lnkHomepage.addSelectionListener(new SelectionAdapter() {
+      @Override
+      public void widgetSelected(SelectionEvent e) {
+        Program.launch("https://hexatomic.github.io/");
+      }
+    });
+
+
+    Link lnkDocumentation = new Link(container, SWT.NONE);
+    lnkDocumentation.setLayoutData(new GridData(SWT.CENTER, SWT.CENTER, false, false, 1, 1));
+    lnkDocumentation.setText("<a>Online User Documentation</a>");
+    lnkDocumentation.addSelectionListener(new SelectionAdapter() {
+      @Override
+      public void widgetSelected(SelectionEvent e) {
+        Program.launch(getOnlineDocumentationUrl());
+      }
+    });
+
+    Link lnkAuthors = new Link(container, SWT.NONE);
+    lnkAuthors.setLayoutData(new GridData(SWT.CENTER, SWT.CENTER, false, false, 1, 1));
+    lnkAuthors.setText("© 2018ff. <a>Hexatomic project team</a>");
+
+
+    lnkAuthors.addSelectionListener(new SelectionAdapter() {
+      @Override
+      public void widgetSelected(SelectionEvent e) {
+        Program.launch("https://github.com/orgs/hexatomic/teams/project/members");
+      }
+    });
+
+    Label lblLicense = new Label(container, SWT.NONE);
+    lblLicense.setLayoutData(new GridData(SWT.CENTER, SWT.CENTER, false, false, 1, 1));
+    lblLicense.setText("Published as Open Source software with the Apache License 2.0");
+
+    return container;
+  }
+
+  /**
+   * Create contents of the button bar.
+   * 
+   * @param parent The parent
+   */
+  @Override
+  protected void createButtonsForButtonBar(Composite parent) {
+    createButton(parent, IDialogConstants.OK_ID, IDialogConstants.OK_LABEL, true);
+  }
+
+  /**
+   * Return the initial size of the dialog.
+   */
+  @Override
+  protected Point getInitialSize() {
+    return new Point(557, 300);
```

Comment:
```suggestion
    return new Point(557, 350);
```

Increase the height of the dialog to show all new content.